frontend/shared/components/src/members/components/view/ViewMemberAccountsBox.vue

```diff
@@ -121,4 +130,40 @@ async function deleteUser(user: User) {
 function isDeletingUser(user: User) {
     return deletingUsers.value.has(user.id)
 }
+
+const securityCode = computed(() => props.member.patchedMember.details.securityCode);
+
+async function renewSecurityCode() {
+    if(!await CenteredMessage.confirm(
+        $t('Nieuwe beveiligingscode'),
```

Comment:
```suggestion
        $t('Wil je de beveiligingscode resetten?'),
```
De titel moet heel expliciet zijn, net zoals de confirmatie knop. De beschrijving mag uitgebreider zijn en dingen verduidelijken.